manim/animation/transform.py

```diff
@@ -357,21 +424,83 @@ def begin(self) -> None:
 
 
 class FadeToColor(ApplyMethod):
+    """Animation that changes color of a mobject.
+
+    Examples
+    --------
+
+    .. manim:: FadeToColor
+        :quality: low
+
+        class FadeToColor(Scene):
+            def construct(self):
+                self.play(FadeToColor(Text("Hello World!"), color=RED))
+
+    """
+
     def __init__(self, mobject: Mobject, color: str, **kwargs) -> None:
         super().__init__(mobject.set_color, color, **kwargs)
 
 
 class ScaleInPlace(ApplyMethod):
+    """Animation that scales a mobject by a certain factor.
+
+    Examples
+    --------
+
+    .. manim:: FadeToColor
+        :quality: low
+
+        class FadeToColor(Scene):
+            def construct(self):
+                self.play(ScaleInPlace(Text("Hello World!"), 2))
+
+    """
+
     def __init__(self, mobject: Mobject, scale_factor: float, **kwargs) -> None:
         super().__init__(mobject.scale, scale_factor, **kwargs)
 
 
 class ShrinkToCenter(ScaleInPlace):
+    """Animation that makes a mobject shrink to center.
+
+    Examples
+    --------
+
+    .. manim:: FadeToColor
+        :quality: low
+
+        class FadeToColor(Scene):
+            def construct(self):
+                self.play(ShrinkToCenter(Text("Hello World!")))
+
+    """
+
     def __init__(self, mobject: Mobject, **kwargs) -> None:
         super().__init__(mobject, 0, **kwargs)
 
 
 class Restore(ApplyMethod):
+    """
+    This functions restores a transformed mobject to its 'last saved state' of the mobject.
+
+    Examples
+    --------
+
+    .. manim:: Restore
+        :quality: low
+
+        class Restore(Scene):
```

Comment:
Circular reference, causes docs failure (same thing for all other examples)
```suggestion
    .. manim:: RestoreExample
        :quality: low

        class RestoreExample(Scene):
```